.gitignore

```diff
@@ -347,3 +347,9 @@ pkg.gitcommit
 
 # Jupyter Notebook Checkpoints
 .ipynb_checkpoints
+
+*.user
+*.obj
+*.tlog
+*.pdb
+Win32/x64/*/*.log
```

Comment:
Can we move those in the beginning, e.g. after CMake and maybe give them a header like `# Windows generated files"?

Reply:
Sure, I can move this.